account_consolidation/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,32 @@
+To install this module, you need the modules `currency_monthly_rate` and
+ to be available in your system.
+
+To configure this module, you need to flag a company as Consolidation in the
+Accounting settings.
+
+Then, you should define a consolidation difference account and a consolidation
+journal on the consolidation company, and create consolidation profiles for
+each subsidiary you want to consolidate.
+
+For each subsidiary to consolidate, make sure the related partner of the
+company has no company_id defined.
+
+Afterwards, you should define a consolidation account from your consolidation
+company, on every active account of the subsidiaries.
+
+You can then use the 'Consolidation : Checks' wizard in Accounting > Adviser to
+ensure every active account of your subsidiaries are set, and company partners
+have no company defined.
+
+Make sure you also defined currency rates and monthly currency rates on the
+currencies used in your subsidiaries, as P&L accounts are consolidated using
+monthly rates and B.S accounts using standard 'spot' rates.
+
+Known issues
+============
+
+* When saving the accounting settings, the group multi-company will be removed
```

Comment:
Is this still the case in v12.0 ?!

Reply:
not very competent at this point, there is a specific test for this right, and specific  hook to remove access group (yml in previous version)
I just kept it without significant changes

Reply:
The issue was that `group_multi_company` is defined in `res.config.settings`, and saving a change in Accounting > Configuration > Settings (e.g. updating consolidation profile) would remove the group for the users who had it defined, if the group is not implied by group employees.

I tried to reproduce on runbot using consolidation manager (I tried to set a rate of 50% on one of the subsidiaries) but I get the following error:

" The operation cannot be completed, probably due to the following:
- deletion: you may be trying to delete a record while other records still reference it
- creation/update: a mandatory field is not correctly set

[object with reference: company_id - company.id] "

So it seems some rights must be fixed to allow this operation.

Reply:
@grindtildeath multicompany settings was moved to general settings so they are not conflicting any more, 
that error caused by not appropriate wizard configuration behavior, I made some improvements of it, and put them in separate commit so we can back-port it to older versions if need